repo2docker/buildpacks/nix/__init__.py

```diff
@@ -57,7 +57,7 @@ def get_assemble_scripts(self):
             ('${NB_USER}', """
             nix-channel --add https://nixos.org/channels/nixpkgs-unstable nixpkgs && \
             nix-channel --update && \
-            nix-shell default.nix --command "command -v jupyter"
+            nix-shell default.nix
```

Comment:
What does this do? Was experimenting here and removed it as I was getting errors that this wasn't being found. So this change is currently empirical and doesn't even fix the problem.

Reply:
Isn't jupyter installed by default by the build pack -> it should always exist?